src/plugins/data_source_management/public/components/data_source_selector/data_source_selector.tsx

```diff
@@ -119,6 +99,16 @@ export class DataSourceSelector extends React.Component<
       this.props.placeholderText === undefined
         ? 'Select a data source'
         : this.props.placeholderText;
+
+    const dataSources = this.props.dataSourceFilter
```

Comment:
Do we have UT to cover this changes? Thanks

Reply:
+1. Approved but I'm wondering if we need UT for this change.

Reply:
yes, we use the same unit test for the filtering which was added when the datasource filter was introduced https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/data_source_management/public/components/data_source_selector/data_source_selector.test.tsx#L137